content/en-us/reference/engine/classes/Accoutrement.yaml

```diff
@@ -30,9 +59,9 @@ deprecation_message: ''
 properties:
   - name: Accoutrement.AttachmentForward
     summary: |
-      Sets the offset position of the object on the Player.
+      The forward vector of the `Class.Weld.C1|C1` of the HeadWeld.
```

Comment:
"HeadWeld" here and as follows does not always apply, for example if the non-default behavior has taken effect and there's no weld by the specific name of "HeadWeld"...